### ISSUE TITLE
the batch size could be set as 30?

### ISSUE THREAD

**Issue by TouchSkyWf** [filipradenovic/cnnimageretrieval-pytorch#35]
if i could set the batch size of my code to be 30?Does this affect network convergence?

**Comment by filipradenovic** [comment]
It might affect. You might need to adjust the learning rate to make the network converge.
